setup_android_deps.sh

```
@@ -0,0 +1,129 @@
+    echo "=== Building LuaJIT for $ABI ($TRIPLET) ==="
+
+    if [ ! -d "$LUAJIT_SRC/src" ]; then
+        echo "ERROR: LuaJIT source not found at $LUAJIT_SRC"
+        echo "Clone it: git clone https://github.com/LuaJIT/LuaJIT.git $LUAJIT_SRC"
+        exit 1
+    fi
+
+    cd "$LUAJIT_SRC"
+    make clean 2>/dev/null || true
+
+    make -j$(nproc) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)